src/ProjectNeon/Assets/Z_Tests/Features/Effects/AttackTests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using NUnit.Framework;
+
+public sealed class AttackTests
+{
+    
+    [Test]
+    public void Attack_ApplyEffect_AttackIsAppliedCorrectly()
+    {
+        Member attacker = TestMembers.Create(
+            s => s.With(StatType.Attack, 1f).With(StatType.MaxHP, 10).With(StatType.Damagability, 1f)
```

Comment:
TestMembers should automatically set `Damagability` to 1, to avoid noisy test duplication.

Reply:
@SilasReinagel I think that we could add an issue of 1 unit to improve all tests, where we implement default values for each `Member` property and make easier to set values so we can tests the program logic. This change would demand the refactor of all tests, task which shouldn't be solved in the scope of #454.